tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -53,14 +59,22 @@ run_aot_tests () {
 if [[ $MODE != "aot" ]];then
     python3 -m venv wasi-env && source wasi-env/bin/activate
     python3 -m pip install -r test-runner/requirements.txt
+
+    # Stress test requires max-threads=8 so it's run separately
+    if [[ -e "${THREAD_INTERNAL_TESTS}spawn_stress_test.wasm" ]]; then 
+        ${IWASM_CMD_STRESS} ${THREAD_INTERNAL_TESTS}spawn_stress_test.wasm
```

Comment:
How do we check if the test failed? We don't seem to check the exit code.

Reply:
Thank you. That's a good point